charts/prometheus-federator/106.0.1-rc.1+up3.0.0-rc.2/values.yaml

```diff
@@ -0,0 +1,242 @@
+      ## this ConfigMap in the deployment namespace
+      subjects:
+        - kind: Group
+          name: system:authenticated
+
+  nameOverride: ""
+
+  namespaceOverride: ""
+
+  replicas: 1
+
+  image:
+    registry: ''
+    repository: rancher/prometheus-federator
+    tag: v3.0.0-rc.2
+    pullPolicy: IfNotPresent
+
+  # Configure how the operator will manage the CustomResourceDefinitions (CRDs) it needs to function.
+  crdManagement:
+    # Enable or disable automatic updates of CRDs during startup.
+    # When true, all CRDs will be updated to the version the operator provides.
+    # When false, only missing CRDs will be installed, and existing ones will not be updated.
+    update: true
+
+    # !! EXPERIMENTAL OPTION !! - Use this feature with caution and careful consideration.
+    # This feature is a stopgap solution and is already expected to be removed in the future.
+    #
+    # Specify whether the operator should detect K3s and RKE2 clusters and exclude `helm-controller` CRDs from management.
+    # When true, `helm-controller` CRDs will not be managed by the operator in these environments, as K3s/RKE2 handle them internally.
+    # When false, the operator will manage `helm-controller` CRDs regardless of the runtime environment.
+    detectK3sRke2: false
+
+  helmController:
+    # Note: should be disabled for RKE2 clusters since they already run Helm Controller to manage internal Kubernetes components
+    enabled: true
+
+    job:
+      image:
+        repository: rancher/klipper-helm
+        tag: v0.9.4-build20250113
+
+  helmLocker:
+    enabled: true
+
+    federate:
+      # Change this to point at all Prometheuses you want all your Project Prometheus Stacks to federate from
+      # By default, this matches the default deployment of Rancher Monitoring
+      targets:
+      - rancher-monitoring-prometheus.cattle-monitoring-system.svc:9090
+
+  # Additional arguments to be passed into the Prometheus Federator image
+  additionalArgs: []
+
+  ## Define which Nodes the Pods are scheduled on.
+  ## ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+
+  ## Tolerations for use with node taints
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: []
+  # - key: "key"
+  #   operator: "Equal"
+  #   value: "value"
+  #   effect: "NoSchedule"
+
+  resources: { }
+    # limits:
+    #   memory: 500Mi
+    #   cpu: 1000m
+    # requests:
+    #   memory: 100Mi
+    #   cpu: 100m
+
+  containerSecurityContext: { }
+    # allowPrivilegeEscalation: false
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # privileged: false
+    # readOnlyRootFilesystem: true
+
+  securityContext: { }
+    # runAsGroup: 1000
+    # runAsUser: 1000
+    # supplementalGroups:
+    # - 1000
+
+  debug: false
+  debugLevel: 0
+
+  cleanup:
+    image:
+      repository: rancher/shell
+      tag: v0.1.19
```

Comment:
We should switch this to Kuberlr based kubectl.

However that should happen in PromFed not charts, so this can wait for now.
I'll make sure to grab this in my PromFed changes.